### PR TITLE
fix: wasm build

### PR DESCRIPTION
**Pull request by hugocaillard**
`reqwest` was not found when building for wasm. Not sure why this regression appeared.

Auto formatted the file

### CODE REVIEW

**Review by obycode — DISMISSED** [review]
lgtm

File: components/clarinet-deployments/Cargo.toml [hunk 1]
```diff
@@ -12,26 +12,36 @@ serde_yaml = "0.8.23"
...
 clarinet_files = { package = "clarinet-files", path = "../clarinet-files", default-features = false }
 chainhook_types = { package = "chainhook-types", path = "../chainhook-types-rs" }
-stacks_rpc_client = { package = "stacks-rpc-client", path = "../stacks-rpc-client", optional = true  }
+stacks_rpc_client = { package = "stacks-rpc-client", path = "../stacks-rpc-client", optional = true }
 
 # CLI
 reqwest = { version = "0.11", default-features = false, optional = true, features = [
     "json",
     "rustls-tls",
 ] }
```
Comment (lgalabru) [inline components/clarinet-deployments/Cargo.toml:21]: if `reqwest` is always used, then we can remove `optional = true` here, and remove `reqwest` from the `wasm` and `onchain` feature gates.

Comment (hugocaillard) [reply components/clarinet-deployments/Cargo.toml:21]: Alright sounds good I'll do that. I wasn't sure why it was optional in the first place

**Review by lgalabru — APPROVED** [review]
Thanks @hugocaillard!

**Pull request merged**